src/java.desktop/unix/classes/sun/awt/X11/MotifColorUtilities.java

```diff
@@ -635,7 +635,8 @@ static void loadMotifDefaultColors(int[] systemColors) {
 
 
     static void loadSystemColors(int[] systemColors) {
-        if (OSInfo.getOSType() == OSInfo.OSType.LINUX) { // Load motif default colors on Linux.
+        if (OSInfo.getOSType() == OSInfo.OSType.LINUX
+            || OSInfo.getOSType() == OSInfo.OSType.BSD) { // Load motif default colors on Linux.
```

Comment:
Update comment maybe? Or remove it, since it does not say anything that the code does not already say...